content/strategy-goals/strategy/code-graph/code-intelligence/index.md

```diff
@@ -79,55 +79,54 @@ Recent key learnings:
 
 ## Strategy and Plans
 
-### FY22 Q4 goals
-
-These tie back to [FY22 Q4 Product/Engineering OKRs](../../../goals/2022_Q4.md#product-engineering)
+### Themes
 
-**Objective: Make the power of our features easier to find and use**
+**Unlock dependency navigation**
 
-- **KR:** Measure and increase WAU for customer facing product teams by 15%. For most teams this will require understanding and improving time-to-value as a driver for increasing active users.
+We believe this is the global code graph’s killer feature. Most developers think in terms of package and dependency versions rather than focusing on external repositories. Supporting precise dependency navigation elevates the code navigation experience to a new level of cross-project analysis. The ability to navigate to any third party dependency a repository references is key to use cases like [Developer onboarding](../use-cases/dev-onboarding.md) where time to value is directly related to how fast a developer is able to navigate and undestand a new codebase (including the dependencies it relies on).
```

Comment:
```suggestion
We believe this is the global code graph’s killer feature. Most developers think in terms of package and dependency versions rather than focusing on external repositories. Supporting precise dependency navigation elevates the code navigation experience to a new level of cross-project analysis. The ability to navigate to any third party dependency a repository references is key to use cases like [Developer onboarding](../../use-cases/dev-onboarding.md) where time to value is directly related to how fast a developer is able to navigate and undestand a new codebase (including the dependencies it relies on).
```